sound/soc/intel/boards/Kconfig

```diff
@@ -678,6 +678,7 @@ config SND_SOC_INTEL_SOUNDWIRE_SOF_MACH
 	select SND_SOC_RT1318_SDW
 	select SND_SOC_RT5682_SDW
 	select SND_SOC_CS42L42_SDW
+	select SND_SOC_CS35L56_SDW
```

Comment:
keep the blank line between SDW & non-SDW

Reply:
> keep the blank line between SDW & non-SDW

Sorry, I didn't get your point. I didn't remove any blank line here.